modules/utxo-lib/src/bitgo/psbt/ProprietaryKeyValUtils.ts

```diff
@@ -0,0 +1,57 @@
+import { KeyValue } from 'bip174/src/lib/interfaces';
+import { decodeProprietaryKey, encodeProprietaryKey } from 'bip174/src/lib/proprietaryKeyVal';
+import { ProprietaryKeySearch, ProprietaryKeyValue } from '../PsbtUtil';
```

Comment:
we have a bit of a import cycle here - can you move these two here and update the import in PsbtUtil?